optaplanner-docs/src/main/docbook/en-US/Chapter-Quick_start/Chapter-Quick_start.xml

```diff
@@ -68,35 +68,87 @@
       </mediaobject>
 
       <para>The simple algorithm used here is the <emphasis>First Fit Decreasing</emphasis> algorithm, which assigns the
-      bigger processes first and assigns the smaller processes to the remaining space. As you can see, it's not optimal,
-      because it does not leave enough room to assign the yellow process D.</para>
+      bigger processes first and assigns the smaller processes to the remaining space. As you can see, it is not optimal, 
+      as it does not leave enough room to assign the yellow process "D".</para>
 
-      <para>Planner does find the more optimal solution fast, by using additional, smarter algorithms. And it scales
-      too: both in data (more processes, more computers) and constraints (more hardware requirements, other
-      constraints). So let's take a look how we can use Planner for this.</para>
+      <para>Planner does find the more optimal solution fast by using additional, smarter algorithms. It also scales: both in data (more processes, more computers) and constraints (more hardware requirements, other
+      constraints). So see how Planner can be used in this scenario.</para>
     </section>
 
     <section xml:id="cloudBalancingProblemSize">
       <title>Problem Size</title>
-
-      <programlisting>2computers-6processes      has   2 computers and    6 processes with a search space of      64.
-3computers-9processes      has   3 computers and    9 processes with a search space of    10^4.
-4computers-012processes    has   4 computers and   12 processes with a search space of    10^7.
-100computers-300processes  has 100 computers and  300 processes with a search space of  10^600.
-200computers-600processes  has 200 computers and  600 processes with a search space of 10^1380.
-400computers-1200processes has 400 computers and 1200 processes with a search space of 10^3122.
-800computers-2400processes has 800 computers and 2400 processes with a search space of 10^6967.</programlisting>
+      <table id="cloudBalancingProblemTable">
+		  <title>Cloud Balancing Problem Size</title>
+		  <tgroup cols='4' align='left' colsep='1' rowsep='1'>
+			<colspec colname='c1'/>
+			<colspec colname='c2'/>
+			<colspec colname='c3'/>
+			<colspec colname='c4'/>
+			<thead>
+				<row>
+					<entry>Problem Size</entry>
+					<entry>Computers</entry>
```

Comment:
Gemma, can you set your editor to use indentation of 2 spaces, instead of a tab (\t)? After merging this PR, I 'll replace all tabs by 2 spaces, so no need to correct this PR.